src/components/TheEnd.jsx

```diff
@@ -51,7 +50,7 @@ const TheEnd = () => {
   const attentioncheck = useSelector(getAttentionCheck);
   const timestamps = useSelector(getTimestamps);
   const status = useSelector(getStatus);
-  const feedback = useSelector(getFeedback);
+  const [feedback, setFeedback] = React.useState("");
```

Comment:
@yahelnachum  is there a reason why you changed the feedback field to being a react state variable and not using useSelector? We had changed the code to move from react state to getting the data right out of redux.

Reply:
I changed it since we don't need the feedback data to persist outside of this page and we were never setting it to begin with. We can move this back to the redux style as long as we actually do set it in there and get it for uploading to aws.